ODPi-Runtime.md

```diff
@@ -25,53 +23,52 @@ Maintenance releases indicate bug fix releases connected to the indicated minor
 Objective
 =========
 
-Objectives of the ODPi TWG is to achieve the following:
+The goals of the ODPi Runtime Specification are:
 
-1.  **For consumers:** ability to run any “ODPi-compatible” software on any “ODPi-compliant” platform and have it work.
+1.  **For End-Users:** Ability to run any Applications on any Platform and have it work.
 
-2.  **For ISVs:** compatibility guidelines that allow them to “test once, run everywhere.”
+2.  **For Software Vendors :** Compatibility guidelines that enable them to ensure thier Applications are interoperable across any Platform.
 
-3.  **For Hadoop platform providers:** compliance guidelines that enable ODPi-compatible software to run successfully on their solutions. But the guidelines must allow providers to patch their customers in an expeditious manner, to deal with emergencies.
+3.  **For Platform Vendors:** Compliance guidelines that enable Applications to run successfully on their Platform. But the guidelines must allow Platform Vendors to patch their End-Users in an expeditious manner, to deal with emergencies.
 
-The goal of this document is to define the interface between ODPi-compliant Apache Hadoop Runtime Services (such as HDFS) and ODPi-compatible applications that achieves the above goal. This interface in turn can be used by ISVs to properly build their software, and will be used as the basis of a compliance test suite that can be used by ODPi-compliant platform providers to test compliance.
+The methodology used in ODPi Runtime Specification is defining the interface(s) between Services on an Platform (such as HDFS) and Applications that achieves the above goals. These interface(s) in turn can be used by Software Vendors to properly build their software, and will be used as the basis of a compliance test suite that can be used by Platform Vendors to test compliance.
```

Comment:
Then even more reason to merge it as-is. The upstream vs. bigtop discussion needs to go on a JIRA and ultimately be conducted live since it is much more nuanced than what GH can handle.